src/components/Pagination/Pagination.scss

```diff
@@ -92,38 +91,6 @@ $stacking-order: (
       transition-duration: duration();
       transition-timing-function: easing();
     }
-    // End - Delete after me when removing newDesignLanguage
-
-    // stylelint-disable selector-max-specificity, selector-max-class
-    &.newDesignLanguage {
```

Comment:
None of this custom styling is needed now that pagination uses buttons and button group